includes/class-piklist-add-on.php

```diff
@@ -218,17 +218,19 @@ public static function is_active($add_on = '')
   public static function current()
   {
     $backtrace = debug_backtrace();
-    
     foreach ($backtrace as $trace)
     {
-      if (strstr($trace['file'], '/parts/'))
+	  $file = isset($trace['file']) ? $trace['file'] : '';
```

Comment:
Up to this point we're not checking to see if `$trace['file']` is set. Is there a reason you felt this was necessary? Seems harmless, as if sets an empty string then the following conditional is false. Are there any issues that could potentially later come up if `$file = ''`?